src/Umbraco.Web/HealthCheck/Checks/Config/TrySkipIisCustomErrorsCheck.cs

```diff
@@ -38,7 +38,7 @@ public override IEnumerable<AcceptableConfiguration> Values
         {
             get
             {
-                var recommendedValue = _serverVersion >= new Version("7.5.0")
+                var recommendedValue = _serverVersion >= new Version("7.5")
```

Comment:
@nul800sebastiaan I've split those translation up now and removed the unneeded key in the latest commit. More work might fall out of this as there are a number of other examples of extra keys being passed into translations but that might be beyond the scope of this bug and justify its own issue being raised.